### PR TITLE
[map] add points on top of map

### PR DESCRIPTION
**Pull request by chejennifer**
- add functionality where if the url contains &ppt=<placeType>, we will plot points for places of that type that are contained within the current map

dev updated (eg. https://dev.datacommons.org/tools/map#%26sv%3DMedian_Age_Person%26pc%3D0%26pd%3DgeoId%2F06027%26pn%3DInyo%20County%2C%20CA%26pt%3DCounty%26ept%3DCounty%26ppt%3DCity)

Screen Shot 2021-09-09 at 11 48 51 PM

### CODE REVIEW

File: server/routes/api/choropleth.py [hunk 1]
```diff
@@ -412,3 +412,45 @@ def choropleth_data(dcid):
...
+    longitude_by_geo = dc_service.get_property_values(geos, "longitude")
+
+    map_points_list = []
+    for geo_id, latitude in latitude_by_geo.items():
+        longitude = longitude_by_geo.get(geo_id, [])
+        if len(latitude_by_geo) == 0 or len(longitude) == 0:
+            continue
+        map_point = {
+            "dcid": geo_id,
+            "name": names_by_geo.get(geo_id, "Unnamed Area"),
```
Comment (beets) [inline server/routes/api/choropleth.py:449]: nit: perhaps "Unnamed place" since these are points

File: static/css/tools/map.scss [hunk 2]
```diff
@@ -235,3 +235,12 @@
...
   padding: 0.3rem;
 }
+
+.dot {
+  r: 4;
+  stroke: var(--dc-gray-lite);
+}
+
+.dot:hover {
+  r: 6;
```
Comment (beets) [inline static/css/tools/map.scss:245]: a slight transition would be nice.

**Review by beets — APPROVED** [review]
nice refactor!

File: server/routes/api/choropleth.py (hunk 1 above)
Comment (chejennifer) [reply server/routes/api/choropleth.py]: done

File: static/css/tools/map.scss (hunk 2 above)
Comment (chejennifer) [reply static/css/tools/map.scss]: done

**Pull request merged**